L1Trigger/TrackFindingTracklet/test/L1TrackNtuplePlot.C

```diff
@@ -2448,8 +2486,7 @@ void L1TrackNtuplePlot(TString type,
                            h2_resVsPt_ptRel_L_99,
                            0,
                            max_pt_ptRel);
-  makeResidualIntervalPlot(
-      type, DIR, "resVsPt_L_pt", h2_resVsPt_pt_L_68, h2_resVsPt_pt_L_90, h2_resVsPt_pt_L_99, 0, max_pt_pt);
+  makeResidualIntervalPlot(type, DIR, "resVsPt_L_pt", h2_resVsPt_pt_L_68, h2_resVsPt_pt_L_90, h2_resVsPt_pt_L_99, 0, 4);
```

Comment:
@vegajustin26 what is this change to 0, 4 ? is this a mistake?

Reply:
No, I don't think it was a mistake. This is for the resolution vs. pT plot on the last two slides (low pT interval), where the y-axis range was especially wide. I remember you told me to have it cut off at 4 GeV, and this was my workaround. Adjusting the previous variable that set the boundary (max_pt_pt) would have also disturbed some other plots, so I just hard-coded it in for this one specific plot.